lxc/delete.go

```diff
@@ -135,7 +135,11 @@ func (c *cmdDelete) Run(cmd *cobra.Command, args []string) error {
 			}
 
 			if ct.Ephemeral {
-				return nil
+				if i == len(resources)-1 {
```

Comment:
just continue will suffice, why do we need the logic wrt to resources?

Reply:
We don't want to call the doDelete function if the instance is ephemeral right ? That's why we return early

Reply:
@tomponline it should be good now